hyp3_gamma/insar/ifm_sentinel.py

```diff
@@ -270,9 +270,18 @@ def insar_sentinel_gamma(reference_file, secondary_file, rlooks=20, alooks=4, lo
     log.info("Starting par_S1_SLC")
     orbit_files = []
     for granule in (reference_file, secondary_file):
-        orbit_file, _ = downloadSentinelOrbitFile(granule)
-        par_s1_slc_single(granule, pol, os.path.abspath(orbit_file))
-        orbit_files.append(orbit_file)
+        try:
+            orbit_file, _ = downloadSentinelOrbitFile(granule)
+        except OrbitDownloadError as e:
+            log.warning(e)
+            log.warning(f'Proceeding using original predicted orbit data included with {granule}')
+            orbit_file = None
+
+        if orbit_file is not None:
+            orbit_file = os.path.abspath(orbit_file)
+            orbit_files.append(orbit_file)
```

Comment:
we want `orbit_files` to be `(None, None)` rather than `[]` when it's passed to `get_product_name()` and `least_precise_orbit_of()`